src/redux/actionTypes/categoriesTypes.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly make sure, the value of  action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       

` export const CHECK_STATUS = bookstore/categories/CHECK_STATUS`